src/main/java/org/sagebionetworks/bridge/sdk/BridgeResearcherClient.java

```diff
@@ -43,4 +44,11 @@ public void signOutUser(String email) {
         
         return get(config.getParticipantsApi(offsetBy, pageSize), ACCOUNT_SUMMARY_PAGED_RESOURCE_LIST);
     }
+    
+    @Override
+    public StudyParticipant getStudyParticipant(String email) {
+        session.checkSignedIn();
+        
+        return get(config.getParticipant(email), StudyParticipant.class);
```

Comment:
Just the readability of repeating that full TypeReference class in order to type with a generic declaration... a simple class reference seems readable as is.